common/djangoapps/static_replace/test/test_static_replace.py

```diff
@@ -633,23 +671,35 @@ def test_canonical_asset_path_with_new_style_assets(self, base_url, start, expec
     def test_canonical_asset_path_with_c4x_style_assets(self, base_url, start, expected, mongo_calls):
         exts = ['.html', '.tm']
         prefix = 'old'
-        c4x_block = 'c4x/a/b/asset'
-        encoded_c4x_block = quote_plus('/' + c4x_block + '/')
+        base_c4x_block = 'c4x/a/b/asset'
+        adjusted_c4x_block = base_c4x_block
```

Comment:
You do this a couple of times where you have two vars `base_...` and `adjusted_...` but they both have the same value,  what's the reason for that?